cursorless-talon/src/actions/actions.py

```diff
@@ -65,13 +65,17 @@ def cursorless_action_or_vscode_command(instruction: dict, target: dict):
             return actions.user.cursorless_vscode_command(value, target)
 
 
-def vscode_command(command_id: str, target: dict, command_options: dict = {}):
+def vscode_command(command_id: str, target: dict, command_options: dict = None):
```

Comment:
a bit odd that the type signature is now technically a lie 🤔.  what do you usually do here @auscompgeek ?

Reply:
I typically change the type to Optional[dict] here. Alternatively I think there's a read-only Mapping type in typing or collections.abc?

Reply:
yeah tbh i'm not too impressed with a lot of the auto-"fixes" from this software.  I think I'll close all these PRs